docfx/doxygen2syntax.cc

```diff
@@ -271,12 +271,13 @@ void AppendEnumSyntax(YAML::Emitter& yaml, YamlContext const& ctx,
 
 void AppendTypedefSyntax(YAML::Emitter& yaml, YamlContext const& ctx,
                          pugi::xml_node node) {
+  auto const aliasof =
+      "<code>" + HtmlEscape(LinkedTextType(node.child("type"))) + "</code>";
```

Comment:
Q: does it matter between `<code></code>` and `<computeroutput></computeroutput>` ?

Reply:
`<computeroutput>` is what Doxygen uses in its XML files.  `<code>` is what we put in (markdown sections of) the YAML files, docfx passes these (unmodified) to its HTML output.  🤯